packaging/chocolatey/crc/crc.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>crc</id>
-    <version>2.11.0</version>
+    <version>2.12.0</version>

```

Comment:
do we need to do every single time when there is a release of crc?

Reply:
yeah this needs to be updated for each release, running the `update.ps1` script will do this automatically

Reply:
wanted to trymore choco/windows specific tooling for updating the `nuspec` but this is better since we are already using similar method